### ISSUE TITLE
Creating a E-Medium category

### ISSUE THREAD

**Issue by gamazeps** [rust-lang/rust#18435]
Hi all,

First : I realize this is not really an issue, but I couldn't think of any better place to put it.

Now to the main idea, we currently have an E-EASY category which is supposed to group many "esay" issues for developpers to start on working on rust, the main idea being that after a few ones they should be able to tackle the harder ones (the not E-EASY ones).

The only problem is that when one try to get out of the easy issues, there is a huge difficulty gap. Indeed the way I felt it is that the first issues help you to get a grasp of how to make rustc work and have a little contact with rust as a language, but in the later issues one have to face a problem that is technically non trivial and face the peculiarities of Rust at the same time, which can be rather hard (at least that's how I felt it).

An idea could be for the more advanced users to tag medium issues as such, so that the less adavanced programmer could grow out of the easy issues. Servo sorta does that with its student projects doing a similar thing with rust could be another good idea.

tldr: once you've done a few E-EASY issues its a huge gap to attack the other issue, having medium difficulty issues would help newcomers to train and be better (and thus more usefull).

I'll dup this on reddit to have it read by more people, don't hesitate to close the issue if you think it's not the place to have it.

**Comment by steveklabnik** [comment]
It seems this discussion has died, and so I'm giving it a close. I don't think we need more tags.
